core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -482,43 +484,37 @@ public void unmount(AlluxioURI path, UnmountOptions options)
   }
 
   /**
-   * Alluxio URIs in the Filesystem don't need a scheme or host. Warn the user and throw an
+   * Checks an {@link AlluxioURI} for scheme and authority information. Warn the user and throw an
    * exception if necessary.
    */
-  protected static void UriCheck(AlluxioURI uri) {
-    boolean logWarning = false;
+  private static void uriCheck(AlluxioURI uri) {
     if (uri.hasScheme()) {
-      if (uri.getScheme().compareTo("alluxio") == 0
-          || uri.getScheme().compareTo("alluxio-ft") == 0) {
-        logWarning = true;
+      if (uri.getScheme().equals(Constants.SCHEME) || uri.getScheme().equals(Constants.SCHEME_FT)) {
```

Comment:
In general, instead of 
```java
if (x || y) { 
  LOG.warn("don't do x or y");
}
```

it's better to do 

```java
if (x) {
  LOG.warn("don't do x");
} else if (y) {
  LOG.warn("don't do y");
}
```

It's a little more code, but more precise and helpful to the user.